setup.py

```diff
@@ -70,7 +70,7 @@ def parse_requirements_file(path):
     zip_safe=False,
     install_requires=parse_requirements_file("requirements.txt"),
     extras_require={"crontrigger": parse_requirements_file("crontrigger_requirements.txt")},
-    python_requires=">=3.8.0,<3.11",
+    python_requires=">=3.8.0,<=3.11",
```

Comment:
```suggestion
    python_requires=">=3.8.0,<3.12",
```

Might probably be a better limit

Reply:
Why? With `=` it's more readable.

Reply:
Just keeping convention how it was